PensionManagementUserLoginService/Models/Context/AuthDbContext.cs

```diff
@@ -0,0 +1,76 @@
+﻿using Microsoft.AspNetCore.Identity;
+using Microsoft.AspNetCore.Identity.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore;
+
+namespace PensionManagementUserLoginService.Models.Context
+{
+    public class AuthDbContext: IdentityDbContext
+    {
+        public AuthDbContext(DbContextOptions<AuthDbContext> options) : base(options)
+        {
+
+        }
+        protected override void OnModelCreating(ModelBuilder builder)
+        {
+            base.OnModelCreating(builder);
+            var readerRoleId = "92327fd8-20fc-42d8-9bd8-536a746bc3ec";
+            var writerRoleId = "ab185359-929b-48ef-85b3-6acb0637dcfd";
+
+            var roles = new List<IdentityRole>
+            {
+                new IdentityRole()
+                {
+                    Id= readerRoleId,
+                    Name="Reader",
+                    NormalizedName="Reader".ToUpper(),
+                    ConcurrencyStamp = readerRoleId
+                },
+                new IdentityRole()
+                {
+                    Id= writerRoleId,
+                    Name="Writer",
+                    NormalizedName="Writer".ToUpper(),
+                    ConcurrencyStamp=writerRoleId
+                },
+            };
+
+            builder.Entity<IdentityRole>().HasData(roles);
+
+
+            var adminUserId = "7ace62fb-9191-4678-946c-3de3f42347da";
```

Comment:
even this ?

Reply:
Generated new Guid Id using visual studio in C# interactive
where it is a unique identifier for the admin